securedrop/create-dev-data.py

```diff
@@ -79,35 +80,31 @@ def main(staging=False):
         db.session.commit()
 
 
-def add_test_user(username, password, otp_secret, is_admin=False,
-                  first_name="", last_name=""):
-    try:
-        user = Journalist(username=username,
-                          password=password,
-                          is_admin=is_admin,
-                          first_name=first_name,
-                          last_name=last_name)
-        user.otp_secret = otp_secret
-        db.session.add(user)
-        db.session.commit()
-        print('Test user successfully added: '
-              'username={}, password={}, otp_secret={}, is_admin={}'
-              ''.format(username, password, otp_secret, is_admin))
-        return user
-    except IntegrityError:
-        print("Test user already added")
-        db.session.rollback()
```

Comment:
During the latest rebase I did, I had to remove this `except` statement causing the function to return `None`, because the rest of the code in this file will crash when `add_test_user()` returns `None`. This was also causing type errors.

For example the code on line 184, 190 and 197 will crash if the journalist is `None`.